x/tradeshield/types/messages_pending_spot_order_test.go

```diff
@@ -100,3 +100,34 @@ func TestMsgDeletePendingSpotOrder_ValidateBasic(t *testing.T) {
 		})
 	}
 }
+
+func TestMsgCancelSpotOrders_ValidateBasic(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  MsgCancelSpotOrders
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: MsgCancelSpotOrders{
+				Creator: "invalid_address",
+			},
+			err: sdkerrors.ErrInvalidAddress,
+		}, {
+			name: "valid address",
+			msg: MsgCancelSpotOrders{
+				Creator: sample.AccAddress(),
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			err := tt.msg.ValidateBasic()
+			if tt.err != nil {
+				require.ErrorIs(t, err, tt.err)
+				return
+			}
+			require.NoError(t, err)
+		})
+	}
+}
```

Comment:
@cryptokage1996 can you add a unit test that test the CLI/msg with a set of pending spot orders